rpcs3/Emu/RSX/GL/GLPipelineCompiler.cpp

```diff
@@ -4,6 +4,11 @@
 
 #include <thread>
 
+namespace utils

```

Comment:
Is sysinfo.h really that heavy that we choose to forward declare this method all over the codebase?